.ci/end2end.groovy

```diff
@@ -43,32 +42,6 @@ pipeline {
             env.APM_UPDATED = isGitRegionMatch(patterns: regexps)
           }
         }
-        dir("${APM_ITS}"){
-          git changelog: false,
-              credentialsId: 'f6c7695a-671e-4f4f-a331-acdce44ff9ba',
-              poll: false,
-              url: "git@github.com:elastic/${APM_ITS}.git"
-        }
-      }
-    }
-    stage('Start services') {
-      options { skipDefaultCheckout() }
-      when {
-        anyOf {
-          expression { return params.FORCE }
-          expression { return env.APM_UPDATED != "false" }
-        }
-      }
-      steps {
-        notifyStatus('Starting services', 'PENDING')
-        dir("${APM_ITS}"){
-          sh './scripts/compose.py start master --no-kibana'
-        }
-      }
-      post {
-        unsuccessful {
-          notifyStatus('Environmental issue', 'FAILURE')
-        }
```

Comment:
the apm-integration-testing is now defined iin the run-e2e.sh script, so this is not required anymore.

Reply:
👍 nice change